app/models/emotion.rb

```diff
@@ -9,6 +9,8 @@
 #  updated_at :datetime         not null
 #
 class Emotion < ApplicationRecord
+  has_many :response, dependent: :destroy
```

Comment:
I think this should be `:responses` -- and a test should have failed to show this error?